src/Features/CSharp/Portable/MoveToResx/CSharpRenameResxService.cs

```diff
@@ -0,0 +1,64 @@
+﻿//// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
We do not want to commit commented out code. Remove until we are ready to move forward with this.